### DISCUSSION TITLE
Add reversed multi-dimension coaster trains

### DISCUSSION THREAD

**Discussion by fidwell** [OpenRCT2/OpenRCT2#17248] [Feature requests] [answered]
Similar to the reversed wooden trains and reversed heartline trains, it would be nice to have a version of the multi-dimension coaster trains that also start reversed. In theory this could be accomplished without any new sprites; just start the trains facing in the opposite direction.

This would be nice for building wing coasters with non-custom objects. If you build a twister coaster with multi-dimension trains, it looks just like a B&M wing coaster, except the guests have to face backwards for the entire ride.

**Comment by ocalhoun6** [comment]
Maybe this could be more generalized, and it could be made possible to start pretty much _any_ ride vehicle facing backwards? Probably just a checkbox in the ride vehicle settings tab.

(And then the special vehicles, like on the wooden or heartline, could be removed, in favor of this option that lets you reverse the cars on any ride.)

Only thing I can think of where it might get a bit weird is in rides that have water splash effects.

**Comment by ZeeMaji** [comment, marked as answer]
Implemented by #19305 as you can use the option to reverse nearly any train.
